addons/binding/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/radiora/handler/RS232Handler.java

```diff
@@ -0,0 +1,130 @@
+import org.openhab.binding.lutron.internal.radiora.RS232Connection;
+import org.openhab.binding.lutron.internal.radiora.config.RS232Config;
+import org.openhab.binding.lutron.internal.radiora.protocol.RadioRACommand;
+import org.openhab.binding.lutron.internal.radiora.protocol.RadioRAFeedback;
+import org.openhab.binding.lutron.internal.radiora.protocol.ZoneMapInquiryCommand;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link RS232Handler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Jeff Lauterbach - Initial contribution
+ */
+public class RS232Handler extends BaseBridgeHandler implements RadioRAFeedbackListener {
+
+    private Logger logger = LoggerFactory.getLogger(RS232Handler.class);
+
+    private RadioRAConnection chronosConnection;
+
+    private RS232Config config;
+    private ScheduledFuture<?> zoneMapScheduledTask;
+
+    RS232Handler(Bridge bridge, RadioRAConnection connection) {
+        super(bridge);
+
+        this.chronosConnection = connection;
+    }
+
+    public RS232Handler(Bridge bridge) {
+        super(bridge);
+
+        this.chronosConnection = new RS232Connection();
+        this.chronosConnection.setListener(this);
+
+    }
+
+    @Override
+    public void dispose() {
+        if (zoneMapScheduledTask != null) {
+            zoneMapScheduledTask.cancel(true);
+        }
+
+        if (chronosConnection != null) {
+            chronosConnection.disconnect();
+        }
+    }
+
+    @Override
+    public void initialize() {
+        connectToChronos();
+
+        scheduleZoneMapQuery();
+    }
+
+    protected void connectToChronos() {
+        this.config = getConfigAs(RS232Config.class);
+        String portName = config.getPortName();
+        int baud = config.getBaud();
+
+        logger.debug("Attempting to connect to Chronos on port {}", portName);
+
+        if (!chronosConnection.open(portName, baud)) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.HANDLER_INITIALIZING_ERROR,
+                    "Error initializing - Failed to connect to chronos");
+            return;
+        }
+
+        logger.debug("Connected successfully");
+
+        updateStatus(ThingStatus.ONLINE);
+    }
+
+    protected void scheduleZoneMapQuery() {
+        logger.debug("Scheduling zone map query at {} second inverval", config.getZoneMapQueryInterval());
+
+        zoneMapScheduledTask = this.scheduler.scheduleAtFixedRate(new Runnable() {
+
+            @Override
+            public void run() {
+                sendCommand(new ZoneMapInquiryCommand());
+            }
+
+        }, 3, config.getZoneMapQueryInterval(), TimeUnit.SECONDS);
+    }
+
+    public void sendCommand(RadioRACommand command) {
+        chronosConnection.write(command.toString());
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        logger.warn("Method handleCommand(...) not implemented");
```

Comment:
What can the user do with this log message, either provide more context or make it debug.